containers/beacon/build.sh

```diff
@@ -0,0 +1,45 @@
+#!/usr/bin/env bash
```

Comment:
This is just a helper script for me to more easily build the Beacon UI locally (basically the commands I was running on the terminal yesterday).